Wireframe/css/style.css

```
@@ -0,0 +1,49 @@
+body{background-color:#fdf0d5;
+    
+} 
+/*git*/
+
+.git-defenation
```

```suggestion
.git-definition
```

I suggest you install a spellchecker in your VSCode. Typos can cause havoc!